src/Compilers/CSharp/Portable/Symbols/Source/LocalFunctionSymbol.cs

```diff
@@ -360,6 +360,8 @@ internal override TypeSymbol IteratorElementType
 
         internal bool IsExpressionBodied => _syntax.Body == null && _syntax.ExpressionBody != null;
 
+        internal override bool IsReadOnly => false;
```

Comment:
Can nested functions read outer `this` without a copy? Perhaps a test?

Reply:
I think using `this` in the local function when `this` is a value type is a compile error.